.env

```diff
@@ -0,0 +1 @@
+AWS_KMS_KEY_ID=__ENV__
```

Comment:
- `__ENV__` is a placeholder
- Actual aws kms key only resides on dev machines (never committed).
- This is enforced by gitignore containing an entry for `.env`